docs/theme_barceloneta.rst

```diff
@@ -0,0 +1,30 @@
+====================================================================
+Creating a Plone theme package based on Barceloneta inside a package
```

Comment:
Please follow our style-guide about headings: https://docs.plone.org/about/contributing/documentation_styleguide.html

Reply:
I don't understand this comment, what should I do here? I copied the header from other doc file like: 

https://raw.githubusercontent.com/plone/bobtemplates.plone/master/docs/theme.rst

Reply:
We have a section about headings in our style-guide, unfortunately this tents to be forgotten.

So at least for new files, I try to remind people about the guidelines and we adjusting slowly already written documentation.

If you for example browse the docs.plone.org you will see that already through the whole documentation.

From our style-guide:

| Headings And Filenames
| Start capitals – capitalization of all words, regardless of the part of speech.